core/src/test/java/org/frankframework/pipes/XmlIfTest.java

```diff
@@ -118,6 +119,48 @@ void someRegexTextTest() throws Exception {
 		assertEquals(pipeForwardElse, pipeRunResult.getPipeForward().getName());
 	}
 
+	@Test
+	@Disabled("broken in the current implementation, see #6963")
```

Comment:
Niet "broken", maar de test is niet zoals jij het verwacht. Misschien beter om de tests te fixen, en dit comment weg te halen. 😃

Reply:
Ja, maar dat willen jullie niet. Ik heb deze test zelf toegevoegd omdat ik van mening ben dat het zo had moeten werken. Dit kan iig worden verplaatst naar de nieuwe `RegExPipe`. Die moet hier iig aan voldoen.